src/modules/news/news.css

```diff
@@ -0,0 +1,27 @@
+
+.newsPanel {
```

Comment:
@reuben-bradley maybe change the text color here so it's not white on silver? Also make the bg and text color for the news article exposed in `skinvars.css`?